### DISCUSSION TITLE
Middle click

### DISCUSSION THREAD

**Discussion by vincehi** [linearmouse/linearmouse#461] [Q&A]
Your application is great! Congratulation to you. I would like to know if you have planned to integrate the detection of the middle click of a magic mouse as well as tap with three fingers on the trackpad for a middle click would be great. 
Thanks

**Comment by lujjjh** [comment]
Thank you!

This feature is currently on my backlog (#215), but I haven't had a chance to work on it yet. You can subscribe to that thread, and any updates will be sent to you.
